lib/atomic_cache/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module AtomicCache
-  VERSION = "1.0.0"
+  VERSION = "0.1.0.rc1"
```

Comment:
I figured a pre-1.0 would be ideal while we kick the tires in our projects for a bit.